sdk/communication/communication-tiering/test/public/getAcquiredNumberLimit.spec.ts

```diff
@@ -23,7 +23,7 @@ describe(`TieringClient - Get Acquired Number Limits`, function () {
 
   it("get acquired number limits", async function () {
     // print all acquire number limits
-    const resourceId = "9d787bd6-07fc-4c7b-8e57-17f1fee41298";
+    const resourceId = env.RESOURCE_ID || "9d787bd6-07fc-4c7b-8e57-17f1fee41298";
```

Comment:
I would suggest avoiding hardcoding this value and simply read from the env vars. In any case, in recordedClient.ts you are already setting up this same GUID as the value for `env.RESOURCE_ID` whenever running in playback mode, which should be more than enough.

Reply:
ok that makes sense